products/BellHybrid/services/audio/ServiceAudio.cpp

```diff
@@ -171,7 +173,7 @@ namespace service
         if (it != retCodes.end()) {
             return std::make_unique<AudioStopResponse>(it->second, it->first);
         }
-
+        bus.sendUnicast(std::make_shared<service::AudioPlaybackStopNotification>(), service::name::evt_manager);
```

Comment:
why do you send it explicitly to event manager?

Reply:
according to current implementation, the temperature compensation algorithm is implemented in EventManager Service